src/containers/BookForm.js

```diff
@@ -1,39 +1,69 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import DropdownButton from 'react-bootstrap/DropdownButton';
 import Form from 'react-bootstrap/Form';
 import Button from 'react-bootstrap/Button';
 import ButtonGroup from 'react-bootstrap/ButtonGroup';
 import Dropdown from 'react-bootstrap/Dropdown';
+import { connect } from 'react-redux';
+import { createBook } from '../actions/index';
 
 class BookForm extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      title: null,
-      category: null,
+      id: '',
+      title: '',
+      category: '',
     };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleSelect = this.handleSelect.bind(this);
+  }
+
+  handleChange(e) {
+    this.setState({
+      title: e.target.value,
+      id: (Math.random() * 500).toFixed(),
+    });
+  }
+
+  handleSelect(evt) {
+    this.setState({
+      category: evt,
+    });
+  }
```

Comment:
- Combine these 2 methods to be one method as mentioned in the requirements, see [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone3/HandleChange.png)

Reply:
Ok thanks